modules/tests/programs/ncmpcpp/ncmpcpp.nix

```
@@ -0,0 +1,69 @@
+{...}: {
+  name = "ncmpcpp-test";
+  nodes = {
+    machine = {
```

```suggestion
  nodes.machine = {
```

(make sure to change the curly bracket pair)